packages/babel-plugin-jsx/README-zh_CN.md

```diff
@@ -321,7 +321,7 @@ const App = {
         <a target="_blank" href="https://www.antdv.com/">
           <img
             width="32"
-            src="https://qn.antdv.com/logo.png"
+            src="https://www.antdv.com/assets/logo.1ef800a8.svg"
```

Comment:
I don't think it's a good idea to use a URL with a file hash.
Suggestion solution:
- Upload the logo to GitHub: https://github.com/vuejs/babel-plugin-jsx/assets/6481596/8d604d42-fe5f-4450-af87-97999537cd21
- Replace with a permanent URL.

Reply:
yeah